controllers/volumereplication_controller.go

```diff
@@ -130,58 +132,58 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		_, pv, err := r.getPVCDataSource(nameSpacedName)
 		if err != nil {
 			_ = r.updateReplicationStatus(instance, replicationv1alpha1.ReplicationFailure, err.Error())
-			r.Log.Error(err, "failed to get dataSource for PVC", "dataSourceName", nameSpacedName.Name)
+			r.Log.Error(err, "failed to get PVC", "PVCName", instance.Spec.DataSource.Name)
```

Comment:
do you want to add instance name here as well?

Reply:
Nope, now we have a instance name by default